CONTRIBUTING.md

```diff
@@ -36,6 +36,25 @@ Coding Standards]. Compliance of your code is checked automatically using
 Travis CI. See [clang-format] and [clang-tidy] configs for more details about
 coding standards.
 
+## How to add an extension
```

Comment:
I think it also makes sense to update build instructions: we probably need to mention that there is a dependency on `SPIRV-Headers` which can be either configured manually or it will be downloaded automatically

Reply:
Done.